seleniumJSHttpEnvTest/src/test/scala/org/scalajs/jsenv/selenium/CamelCaseTest.scala

```diff
@@ -0,0 +1,10 @@
+package org.scalajs.jsenv.selenium
+
+import org.junit.Assert._
+import org.junit.Test
+
+class CamelCaseTest {
```

Comment:
I didn't notice it before: But I think we should give this a name that describes what it tests, not what it does.

So maybe: `ModuleSupportTest` and call the test `testBasicImport`. (notice that different casing, according to Scala.js style).

The rationale here is that failures are easier to understand. Compare:

"failed: CamelCaseTest" to "failed: ModuleSupportTest".

In the second case, it hints that there is a problem with module support (which might be the case). In the first case, it hints that there is a problem with camel casing, which isn't a functionality provided by this repo.